### ISSUE TITLE
Add config flags for common libs which need Webpack config

### ISSUE THREAD

**Issue by insin** [insin/nwb#108]
`enzyme` and `sinon` for starters.

**Comment by insin** [comment]
How about `webpack.compat` config which makes all the necessary config tweaks for you?

```js
module.exports = {
  webpack: {
    compat: {
      enzyme: true,
      sinon: true
    }
  }
}
```

I was also considering scanning `package.json` and automtically adding tweaks for known-incompatible libraries, but it seems a bit too magical.\

The version of React also affects which tweaks you need to make for enzyme, so we might need to scan `package.json` anyway.

**Comment by insin** [comment]
Implemented in 388e8bf6e84e41fe8fb91b73f9531c6dec4ed4f1
